views/components/Matrix.d3.js

```diff
@@ -60,7 +57,7 @@ export default class Matrix {
       .select(this.el)
       .append('svg')
       .attr('width', () => '100%')
-      .attr('height', () => '100%')
+      .attr('height', () => data.length * this.cardSize + margin.top + margin.bottom)
```

Comment:
I'm curious how this number came about, also what do you think about moving it to a constant.
```suggestion
      const calculatedHeight = data.length * this.cardSize + margin.top + margin.bottom
      
      
      .attr('height', () => calculatedHeight)
```

Reply:
Non-fixed heights were behaving weirdly when I added multiple matrices, so I calculated by the cell height.